parquet-variant/src/variant.rs

```diff
@@ -1063,6 +1064,34 @@ impl<'m, 'v> Variant<'m, 'v> {
             _ => None,
         }
     }
+
+    /// Return a new Variant with the path followed.
+    ///
+    /// If the path is not found, `None` is returned.
+    pub fn get_path(&self, path: &VariantPath) -> Option<Variant> {
+        let mut output = self.clone();
+        for element in path.deref() {
```

Comment:
It's not usually necessary to invoke `deref` directly?

Reply:
... but apparently this is a context where deref isn't automatic (tried it in playground). Tho we could perhaps be more clear by 
```suggestion
        for element in path.iter() {
```
... which _does_ deref automatically?